src/backend/joanie/core/api/admin.py

```diff
@@ -308,3 +308,35 @@ class ContractDefinitionViewSet(viewsets.ModelViewSet):
     permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
     serializer_class = serializers.AdminContractDefinitionSerializer
     queryset = models.ContractDefinition.objects.all()
+
+
+class OrderGroupViewSet(
```

Comment:
Instead of retrieve `product_id` through kwargs in every function, we could image to centralize it. e.g : by using a mixin.

We could create a mixin `NestedRelatedObjectMixin` which would add a method to retrieve the nested object id from kwargs and in charge to bind this id into the context passed to the serializer and override methods to retrieve objet (get_object and get_queryset) ?

Class attributes could allow us to define `nested_lookup_field` and `nested_lookup_url_kwarg` 🤔